scout/parse/case.py

```diff
@@ -473,12 +474,18 @@ def parse_case(config):
         raise ConfigError("A case has to have a 'family'")
 
     individuals = parse_individuals(config["samples"])
+    synopsis = None
+    if config.get("synopsis"):
+        synopsis = ". ".join(config["synopsis"])
```

Comment:
I'm separating the lists in synopsis with ". " so that everything is written in synopsis as different sentences. But I'm open to suggestions here!

Reply:
Mmm I don't know, doesn't feel very intuitive having the synopsis field accepting a list of strings in Scout. Think also about other external users uploading their cases. Perhaps we could fix these lists to be concatenated already in cg so that the synopsis is a string when the config file is created?

Reply:
I've modified it to accept both strings and lists. If it's a list it joins the strings with ". " in between.